ecommerce/factories.py

```diff
@@ -87,3 +94,36 @@ class CoursePriceFactory(DjangoModelFactory):
 
     class Meta:  # pylint: disable=missing-docstring,no-init,too-few-public-methods,old-style-class
         model = CoursePrice
+
+
+class CouponFactory(DjangoModelFactory):
+    """Factory for Coupon"""
+    coupon_code = FuzzyText()
+    amount_type = 'percent-discount'
+    amount = FuzzyDecimal(0, 1)
+    num_coupons_available = FuzzyInteger(0, 15)
+    num_redemptions_per_user = FuzzyInteger(0, 5)
```

Comment:
Is `0` a reasonable value for these fields? If not, it shouldn't be included in the FuzzyInteger range.